.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git
+.build
+.dockerignore
+Dockerfile
```

Comment:
This is the current `.gitignore` converted to Docker ignore syntax, maybe add the ones not currently present? ☺️ 

```
.git/
**/.DS_Store
.build
Packages
*.xcodeproj
```